src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/CertificateRevocationListBuilder.cs

```diff
@@ -200,15 +200,13 @@ public static CertificateRevocationListBuilder LoadPem(string currentCrl, out Bi
 
         public static CertificateRevocationListBuilder LoadPem(ReadOnlySpan<char> currentCrl, out BigInteger currentCrlNumber)
         {
-            ReadOnlySpan<char> source = currentCrl;
-
-            while (PemEncoding.TryFind(source, out PemFields fields))
+            foreach ((ReadOnlySpan<char> contents, PemFields fields) in new PemEnumerator(currentCrl))
             {
-                if (source[fields.Label].SequenceEqual("CRL"))
+                if (contents[fields.Label].SequenceEqual("CRL"))
```

Comment:
At least for what OpenSSL produces...

```suggestion
                if (contents[fields.Label].SequenceEqual("X509 CRL"))
```

Reply:
https://datatracker.ietf.org/doc/html/rfc7468#section-6 agrees.

Told ya I hadn't tested this one yet :smile: